src/components/DataTable/components/Cell/tests/Cell.test.tsx

```diff
@@ -57,12 +53,12 @@ describe('<Cell />', () => {
         />,
       );
 
-      expect(cell.prop('aria-sort')).toBe(sortDirection);
+      expect(cell.find('th').prop('aria-sort')).toBe(sortDirection);
```

Comment:
Should we search based upon the name of the react component instead of the dom element it renders as?

```
expect(cell.find(Cell).prop('aria-sort')).toBe(sortDirection);
```

Reply:
We would be asserting that the prop we passed to the component has the prop we passed it (which will be true 100% of the time). Now we're asserting the prop is placed on the `th` which was the original intent of the test 😄

Reply:
Groovy 👍